apps/dotcom/src/components/DocumentName/DocumentName.tsx

```diff
@@ -39,6 +39,9 @@ const MAX_TITLE_WIDTH_PX = 420
 const BUTTON_WIDTH = 44
 const STYLE_PANEL_WIDTH = 148
 const MARGIN_BETWEEN_ZONES = 12
+const SMALL_SCREEN_WIDTH = 841
```

Comment:
thx @Taha-Hassan-Git!
actually, i might ask you to make this change so that this number doesn't get of sync with our canonical constant:
`PORTRAIT_BREAKPOINTS[PORTRAIT_BREAKPOINT.TABLET]`

Reply:
I can't do that because of a lint rule around importing from our own package. Having to use the useBreakpoint hook.